test/disabled/Hog.C

```diff
@@ -735,7 +735,8 @@ DEFINE_STORAGE_GROUP(
   hobbes::storage::Reliable,
   hobbes::storage::ManualCommit
 );
-
+#if 0
+// todo: smunix: waitSegFileGone() is broken
```

Comment:
is it because of the test being unstable?  I did a few runs locally (using master branch) and the test passed in a row.